src/games/supported/BreakoutSuper.hpp

```diff
@@ -0,0 +1,87 @@
+/* *****************************************************************************
+ * The lines 68 and 76 are based on Xitari's code, from Google Inc.
```

Comment:
I added this line for the lives signal in the games in which DeepMind provided xitari code. Super Breakout is not one of them.